grafana-plugin/src/containers/IntegrationContainers/IntegrationTemplatesList.tsx

```diff
@@ -40,6 +40,8 @@ const IntegrationTemplateList: React.FC<IntegrationTemplateListProps> = ({
   const [showConfirmModal, setShowConfirmModal] = useState(false);
   const [autoresolveValue, setAutoresolveValue] = useState<boolean>(alertReceiveChannelAllowSourceBasedResolving);
 
+  const store = useStore();
```

Comment:
In line 37 we already destructure from `useStore`, can we add declaration of `features` there instead of creating another const?

One more note is that this component depends on store but is not wrapped with `observer`, probably we should add it